src/msg.cpp

```diff
@@ -86,8 +86,7 @@ int zmq::msg_t::init ()
     u.vsm.flags = 0;
     u.vsm.size = 0;
     u.vsm.group[0] = '\0';
-    u.vsm.routing_id = 0;
-    u.vsm.fd = retired_fd;
+    u.vsm.routing_id = 0;    
```

Comment:
there's trailing whitespace here and in other places below - want to fix before merge? or in a new PR?